crates/cargo-util-schemas/src/manifest/mod.rs

```diff
@@ -157,9 +167,12 @@ pub struct InheritablePackage {
 /// tables.
 #[derive(Deserialize, Serialize, Clone, Debug)]
 #[serde(rename_all = "kebab-case")]
+#[cfg_attr(feature = "unstable-schema", derive(schemars::JsonSchema))]
 pub struct TomlPackage {
     pub edition: Option<InheritableString>,
+    #[cfg_attr(feature = "unstable-schema", schemars(with = "String"))]
     pub rust_version: Option<InheritableRustVersion>,
```

Comment:
These attributes aren't acknowledging the `None` case

Reply:
fixed this for more types